mathics/builtin/box/image.py

```diff
@@ -1,6 +1,25 @@
 # -*- coding: utf-8 -*-
 
+import base64
```

Comment:
Without a docstring at the top, this doesn't appear in any printed documentation or in Django docs. If that is intended, then the homegrown tagging such as `ImageBox` (on line 26) is not used. 

We should decide way we want to go and either add a docstring at the top or remove the homegrown tagging.